src/ppx_import.ml

```diff
@@ -440,13 +455,14 @@ let rec psig_of_tsig ~subst (tsig : Compat.signature_item_407 list) =
             ; oval_prims = []
             ; oval_attributes = [] }
         in
-        let oval = Primitive.print p oval in
+        let oval = Ocaml_common.Primitive.print p oval in
         oval.Ot.oval_prims
```

Comment:
I'm definitely not familiar with that either 😅 But looking at the [source code](https://github.com/ocaml/ocaml/blob/005ba108dbe9f14a07175481f7aa8501571f96e3/typing/primitive.ml#L159-L164), it can be either `[p.prim_name; p.prim_native_name]` or `[p.prim_name]` depending on whether `p.prim_native_name` is an empty string.

Does that make sense ?

Reply:
It does, thanks! What would you think about inlining those 4 lines you've linked to? That way `ppx_import` wouldn't use `Outcometree` anymore.

Reply:
Although I have to admit that that only indirectly has to do with the port to `ppxlib` in the sense that one of the advantages `ppxlib` tries to achieve is that PPXs don't need to use the unstable `compiler-libs` anymore.

Reply:
Thanks, that should be good now 🥳💪